README.md

```diff
@@ -2,13 +2,10 @@
   <img src="assets/header.gif"/><br>
   <img src="https://github.com/tuist/tuist/workflows/Tuist/badge.svg" alt="Tuist">
   <img src="https://img.shields.io/github/v/release/tuist/tuist?include_prereleases&style=flat-square" alt="Latest Version">
-  <a href="http://community.tuist.io"><img src="https://img.shields.io/badge/community-tuist-blue?style=flat-square" alt="Community"></a>
-  <img src="https://img.shields.io/github/forks/tuist/tuist?style=flat-square" alt="GitHub forks">
-  <img src="https://img.shields.io/github/stars/tuist/tuist?style=flat-square" alt="GitHub stars">
   <img src="https://img.shields.io/github/commit-activity/w/tuist/tuist?style=flat-square" alt="Commit Activity">
-    <img src="https://img.shields.io/github/contributors/tuist/tuist?style=flat-square" alt="Contributors">
   <img src="https://img.shields.io/github/commits-since/tuist/tuist/latest?style=flat-square" alt="Latest Commits">
   <a href="http://twitter.com/tuistio"><img src="https://img.shields.io/twitter/follow/tuistio?style=flat-square" alt="Twitter Followers"></a>
+  <a href="https://join.slack.com/t/tuistapp/shared_invite/zt-g38gajhj-D6LLakrPnVCy4sLm24KxaQ"><img src="https://img.shields.io/badge/slack-tuist-green.svg?logo=slack" alt="Slack Workspace">
```

Comment:
There was no link to slack in the Readme, should we make it even more relevant (e.g. mentioning it somewhere below)?